src/util/codecs.js

```diff
@@ -186,3 +186,25 @@ export const codecsForPlaylist = function(master, media) {
 export const isLikelyFmp4Data = (bytes) => {
   return findBox(bytes, ['moof']).length > 0;
 };
+
+/*
+ * Check if a codec string refers to an audio codec.
+ *
+ * @param {String} codec codec string to check
+ * @return {Boolean} if this is an audio codec
+ * @private
+ */
+export const isAudioCodec = function(codec) {
```

Comment:
brought these back from 1.x

Reply:
not as part of this PR, but may make sense to move this into vhs-utils.